codelists/codelists.txt

```diff
@@ -1 +1,6 @@
 opensafely/alanine-aminotransferase-alt-tests/2298df3e
```

Comment:
There's a new version of this codelist https://www.opencodelists.org/codelist/opensafely/alanine-aminotransferase-alt-tests/11d2e678/